tests/containers/bci_prepare.pm

```diff
@@ -81,6 +81,35 @@ sub activate_virtual_env {
     assert_script_run('source bci/bin/activate');
 }
 
+sub scc_credentials_host {
+    # Non-RMT SCC credentials for Non-suse host running BCI tests
+    my ($runtime, $scc_reg_code) = @_;
+    record_info('Credentials', "check");
+    die("Input parameter missing: $runtime, $scc_reg_code.") unless ($runtime && $scc_reg_code);
+    my $credentials = "/etc/zypp/credentials.d/SCCcredentials";
+    my $credentials_tmp = "/tmp/SCCcredentials";
+    my @commands = (
+        # Check registry connection is ok
+        "curl -ILks -m 60 -o /dev/null registry.suse.com",
+        # Collect credentials, in a local temporary file, from sle image
+        "$runtime run --rm registry.suse.com/suse/sle15:latest " .
```

Comment:
I don't understand what's going on here. Can you elaborate?

Reply:
Please simply read the comments on each @command, executed in sequence.
It performs what explained in the PR header

Reply:
I think this whole route via `SUSEConnect` is unnecessary and we can just directly create the `/etc/zypp/credentials.d/SCCcredentials` file here.

That also means we don't need the `sle15` container as a test dependency.

Reply:
This is waht this PR does, based on the above named documentation, 
**automatically** gets the credentials from an existing sle-container and store the content into a temporary file;
then checks presence of `password` parameter for validation. 
Then that file is moved into the /etc/zypp/credentials.d/SCCcredential.
The last test is an extra that I can remove.

Reply:
I see in you output named the product `SLE 15-SP6`, but the container is What kind of code did you use? I used one from container hosts tests and the command above is ok.

Reply:
It doesn't work with a 12-SP5 LTSS code.